spec/factories/prior_authority_applications.rb

```diff
@@ -89,6 +90,20 @@
         'file_path' => '123123123'
       }
     end
+
+    trait :per_hour do
+      cost_type { 'per_hour' }
+      cost_per_hour { '80.0' }
+      cost_per_item { nil }
+      item_type { nil }
+      period { 420 }
+    end
+
+    trait :no_travel do
+      travel_time { 0 }
```

Comment:
Because the travel detail form is skipped by default unless the user clicks through to it from the summary screen, I think travel_time should never be 0 - either the screen is skipped, so it's nil, or the form is filled out, in which case validations force it to be > 0. So I think this in unrealistic data.

Reply:
👍 will update